packages/nouns-webapp/src/components/VoteModal/index.tsx

```diff
@@ -204,6 +201,18 @@ const VoteModal = ({ show, onHide, proposalId, availableVotes }: VoteModalProps)
           >
             {isLoading ? <Spinner animation="border" /> : <Trans>Submit Vote</Trans>}
           </Button>
+
+          <div
+          className={classes.gasFreeVotingWrapper}
+          >
+            <span
+            className={classes.gasFreeVotingCopy}
+            >
+              <Trans>
+              Gas spent on voting will be refunded to you
```

Comment:
Worth mentioning that gas will be refunded to the direct caller in the current version? If voting from a Gnosis Safe, the refund will go to the safe.